airbyte-cdk/python/unit_tests/sources/file_based/scenarios/scenario_builder.py

```diff
@@ -41,6 +41,7 @@ def __init__(
             expected_discover_error: Tuple[Optional[Type[Exception]], Optional[str]],
             expected_read_error: Tuple[Optional[Type[Exception]], Optional[str]],
             incremental_scenario_config: Optional[IncrementalScenarioConfig],
+            file_write_options: Dict[str, Any]
```

Comment:
I had to add some new boilerplate to the scenario builder since now we need to pass some format args to the CSV writer specifically. I would imagine other file type writers might also need to do the same once we get to them